packages/services/f-http/package.json

```diff
@@ -31,6 +31,7 @@
     "build": "vite build",
     "lint": "eslint \"!(dist)/**/*.{js,vue}\"",
     "lint:fix": "yarn lint --fix",
+    "storybook:build": "vite build",
```

Comment:
Could this potentially hit a race condition with the `storybook:build` script being run in the storybook package?

I'm guessing you need this to complete before that other script builds?